content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/android.md

```diff
@@ -57,7 +57,7 @@ dependencies {
 
 ### Specify application details in the UI
 
-1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][2].
+1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][2].
```

Comment:
```suggestion
1. Navigate to [**UX Monitoring** > **Setup & Configurations** > **New Application**][2].
```
Just an FYI I received a message "You need the
RUM Apps Write permission to view this page."
So maybe we need a Note here just to mention this ?

Reply:
It should be okay for the user who would be following these steps, but I appreciate you flagging it!